packages/firebase_ml_vision/test/firebase_ml_vision_test.dart

```diff
@@ -22,14 +21,672 @@ void main() {
         switch (methodCall.method) {
           case 'TextDetector#detectInImage':
             return returnValue;
+          case 'BarcodeDetector#detectInImage':
+            return returnValue;
           default:
             return null;
         }
       });
       log.clear();
     });
 
-    group('$BarcodeDetector', () {});
+    group('$BarcodeDetector', () {
+      test('detectInImage unknown', () async {
+        final Map<dynamic, dynamic> _barcode = <dynamic, dynamic>{
+          'raw_value': 'hello:raw',
+          'display_value': 'hello:display',
+          'value_type': 0,
+          'format': 0,
+          'left': 1,
+          'top': 2,
+          'width': 3,
+          'height': 4,
+          'points': <dynamic>[
+            <dynamic>[5, 6],
+            <dynamic>[7, 8],
+          ],
+        };
+
+        returnValue = <dynamic>[_barcode];
+
+        final BarcodeDetector detector =
+            FirebaseVision.instance.barcodeDetector();
+        final FirebaseVisionImage image =
+            new FirebaseVisionImage.fromFilePath('empty');
+
+        final List<VisionBarcode> barcodes =
+            await detector.detectInImage(image);
+
+        expect(log, <Matcher>[
+          isMethodCall(
+            'BarcodeDetector#detectInImage',
+            arguments: 'empty',
+          ),
+        ]);
+
+        final VisionBarcode barcode = barcodes[0];
+        expect(barcode.boundingBox, const Rectangle<num>(1, 2, 3, 4));
```

Comment:
This should be `Rectangle<int>`, right? Can you change all the `Point<num>` to `Point<int>` as well?

Reply:
Yes.  I referred TextDetector test. (https://github.com/flutter/plugins/blob/master/packages/firebase_ml_vision/test/firebase_ml_vision_test.dart#L100)